src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -4950,6 +5154,84 @@ namespace HVACVariableRefrigerantFlow {
+                                               RoutineName);
+                        VRFTU(VRFTUNum).SuppHeatCoilFluidMaxFlow = VRFTU(VRFTUNum).SuppHeatCoilFluidMaxFlow * rho;
+                    }
+                } else if (VRFTU(VRFTUNum).SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingSteam) {
+                    // steam supplemental heating coil
+                    errFlag = false;
+                    PlantUtilities::ScanPlantLoopsForObject(VRFTU(VRFTUNum).SuppHeatCoilName,
+                                                            TypeOf_CoilSteamAirHeating,
+                                                            VRFTU(VRFTUNum).SuppHeatCoilLoopNum,
+                                                            VRFTU(VRFTUNum).SuppHeatCoilLoopSide,
+                                                            VRFTU(VRFTUNum).SuppHeatCoilBranchNum,
+                                                            VRFTU(VRFTUNum).SuppHeatCoilCompNum,
+                                                            errFlag,
+                                                            _,
+                                                            _,
+                                                            _,
+                                                            _,
+                                                            _);
+                    if (errFlag) {
+                        ShowFatalError(RoutineName + ": Program terminated for previous conditions.");
+                    }
+                    VRFTU(VRFTUNum).SuppHeatCoilFluidMaxFlow = SteamCoils::GetCoilMaxSteamFlowRate(VRFTU(VRFTUNum).SuppHeatCoilIndex, ErrorsFound);
+                    if (VRFTU(VRFTUNum).SuppHeatCoilFluidMaxFlow > 0.0) {
+                        int SteamIndex = 0; // fluid type index of 0 is passed if steam
+                        Real64 TempSteamIn = 100.0;
+                        Real64 SteamDensity = FluidProperties::GetSatDensityRefrig(fluidNameSteam, TempSteamIn, 1.0, SteamIndex, RoutineName);
+                        VRFTU(VRFTUNum).SuppHeatCoilFluidMaxFlow = VRFTU(VRFTUNum).SuppHeatCoilFluidMaxFlow * SteamDensity;
+                    }
+                }
+                // fill fluid outlet node for steam coil SuppHeatCoilFluidOutletNode
+                VRFTU(VRFTUNum).SuppHeatCoilFluidOutletNode = PlantLoop(VRFTU(VRFTUNum).SuppHeatCoilLoopNum)
+                                                                  .LoopSide(VRFTU(VRFTUNum).SuppHeatCoilLoopSide)
+                                                                  .Branch(VRFTU(VRFTUNum).SuppHeatCoilBranchNum)
+                                                                  .Comp(VRFTU(VRFTUNum).SuppHeatCoilCompNum)
+                                                                  .NodeNumOut;
+                MySuppCoilPlantScanFlag(VRFTUNum) = false;
+            } else { // pthp not connected to plant
```

Comment:
Let me guess, you copied this from PTHP!

Reply:
I think so. I added steam and hot water supplemental heating coils to  PTHP a few years back.